conf/config-default.yaml

```diff
@@ -185,6 +185,7 @@ plugins:                          # plugin list
   - proxy-cache
   - proxy-mirror
   - request-id
+  - sls-logger
```

Comment:
Is this plugin only for aliyun SLS?

Reply:
yes

Reply:
Should the name show aliyun?

Reply:
sls is the name of aliyun log service, https://www.aliyun.com/product/sls/